ignite/cmd/plugin.go

```diff
@@ -27,18 +30,27 @@ const (
 // If no configuration found, it returns w/o error.
 func LoadPlugins(ctx context.Context, rootCmd *cobra.Command) error {
 	// NOTE(tb) Not sure if it's the right place to load this.
-	chain, err := newChainWithHomeFlags(rootCmd)
+	cfg, err := parseLocalPlugins(rootCmd)
 	if err != nil {
-		// Binary is run outside of an chain app, plugins can't be loaded
+		// if binary is run where there is no plugin.yml, don't load
 		return nil
 	}
-	plugins, err = plugin.Load(ctx, chain)
+
+	// TODO: parse global config
```

Comment:
Why?

Reply:
Why parse global config?

The plan going forward will be

`ParseLocalPluginsConfig` -> `ParseGlobalPluginsConfig` -> `Sort` -> `LoadAllPlugins`

Reply:
Oh! I misunderstood thinking that it referred to the Ignite config instead of the global plugins config 👍